### PR TITLE
Added Calculator Components

### PR DESCRIPTION
**Pull request by Mhdez221993**
Project requirements

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components create a new Calculator.js file.
- In Calculator.js, create a React component that matches this design:
- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class-based components.
- Once the Calculator is ready, import it and use it in your main component (App.js).
- Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,17 +1,9 @@
-import Typography from '@material-ui/core/Typography';
-import './App.css';
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (vichuge) [inline src/App.js:3]: - [ ] Please, consider using class-based for App.js component in place function. In the next steps, you'll see why this is important. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Mhdez221993) [reply src/App.js:3]: Thank you, I missed that one.

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi @Mhdez221993 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**